dockerfiles/Dockerfile-deb-builder

```diff
@@ -78,7 +78,8 @@ RUN curl --proto '=https' --tlsv1.2 -sSf -o /tmp/rustup-init \
   https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-gnu/rustup-init \
   && chmod +x /tmp/rustup-init \
   && /tmp/rustup-init -y --default-toolchain "${RUST_VERSION}" --profile minimal \
-  && rm /tmp/rustup-init
+  && rm /tmp/rustup-init \
+  && rustup update
```

Comment:
no cargo will install whatever the toolchain file says :o the reason I do a `rustup update` is that IIUC cargo understand `rust-toolchain.toml` files since a more recent version